test/resources/test_secure_messaging.py

```diff
@@ -11,12 +11,12 @@
 with open('test/test_data/secure_messaging/message.json') as json_data:
     message = json.load(json_data)
 url_get_messages = '{}{}{}'.format(app.config['RAS_SECURE_MESSAGING_SERVICE'],
-                                   'messages', '?label=INBOX&limit=1000')
+                                   'v2/messages', '?label=INBOX&limit=1000')
```

Comment:
🎨 Could be rewritten to f"{app.config['RAS_SECURE_MESSAGING_SERVICE']}v2/messages?label=INBOX&limit=1000" .  Feel free to not bother though ^^, at some point there needs to be a pass over the whole repo to make this look tider